Cargo.toml

```diff
@@ -10,5 +10,5 @@ build = "build.rs"
 [dependencies]
 libc = "*"
```

Comment:
For an eventual cargo upload, it would also make sense to specify libc = "0.2.6" as "*" is deprecated now, too.